Add installation paragraph for pip

damian0o opened the pull request:
As a developer i would like have to have simple installation command close at hand.

rycus86 commented:
Thanks for this!
Do you think it's also worth mentioning that you can just add it to `requirements.txt` if you already use that or have IDE integration for it, like in PyCharm for example?

damian0o commented:
I am quite not sure how it should be described with or without specific version so i added both.

Added command for installing dependencies from requirements.txt file as well - but here there are also many alternatives :)
 ```
pip install -r requirements.txt
or
python -m pip install -r requirements.txt
```

rycus86 commented:
Thanks a lot again for this!
